akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/BehaviorTestKitImpl.scala

```diff
@@ -172,6 +173,21 @@ private[akka] final class BehaviorTestKitImpl[T](
   override def clearLog(): Unit = context.clearLog()
 
   override def receptionistInbox(): TestInboxImpl[Receptionist.Command] = context.system.receptionistInbox
+
+  override def stopSelf: Boolean = {
```

Comment:
I don't think there is a point in the `Boolean` here since it will always return `true` (!isAlive).
The signature could be
```
def stopSelf(): Unit
```

or simply
```
def stop(): Unit
```